pkg/stream/edgedmetricsconnection.go

```diff
@@ -62,6 +62,12 @@ func (ms *EdgedMetricsConnection) Serve(tunnel SafeWriteTunneler) error {
 		return err
 	}
 	req.Header = ms.Header
+	// Since current tunnel implementation only support Text message,
+	// we should force Accept-Encoding to identity to avoid any compression.
+	// For example, user may pass accept-encoding: gzip in header.
+	// TODO: luogangyi
+	// When we support binary message, we can remove this setting.
+	req.Header.Set("accept-encoding", "identity")
```

Comment:
User's request header is `ms.Header`, not need to specify again.

Reply:
this is used to change user's header setting, because we are not support compressed body.
You can try
```
curl  -k -H "Accept-Encoding: gzip" https://edgenode1:10350/metrics|gunzip
```
it returns unexpected EOF

so, we have to force Accept-Encoding to identity

Reply:
I think it is not a common usage, most users use prometheus directly.

Does k8s support it?

Reply:
In fact, this problem is prometheus brings.
I find prometheus will add "Accept-Encoding: gzip" in request...

Reply:
> In fact, this problem is prometheus brings.
> I find prometheus will add "Accept-Encoding: gzip" in request...

Indeed it is，How to configure when use prometheus directly?

Reply:
> In fact, this problem is prometheus brings.
> I find prometheus will add "Accept-Encoding: gzip" in request...

Indeed it is，How to configure when use prometheus directly?

Reply:
“Indeed it is，How to configure when use prometheus directly?” hi, has this done? i'm a fresh-hand in the prometheus and kubeedge area, i am wondering how to configure using prometheus to monitor edgenode as k8s worker node do. any help would appreciate.